SRT/CDB/MACI/Containers/WeatherStationContainer/WeatherStationContainer.xml

```diff
@@ -7,5 +7,5 @@
 	<Autoload>
 		<cdb:_ string="baci" />
 	</Autoload>
-	<LoggingConfig centralizedLogger="Log" minLogLevel="2" minLogLevelLocal="2" dispatchPacketSize="0" immediateDispatchLevel="8" flushPeriodSeconds="1" />
+	<LoggingConfig centralizedLogger="Log" minLogLevel="5" minLogLevelLocal="5" dispatchPacketSize="0" immediateDispatchLevel="8" flushPeriodSeconds="1" />
```

Comment:
Be aware that we are bringing some other modifications not strictly related to this issue from master to stable since this branch was created starting from the master branch.